tests/artifacts/test_artifacts.py

```diff
@@ -466,3 +466,19 @@ def test_inline_body():
     )
     assert artifact.spec.get_body() == "123"
     assert artifact.metadata.key == "y"
+
+
+def test_import_artifact_using_relative_path():
+    project = mlrun.new_project("inline", context=str(assets_path()), save=False)
+
+    # log an artifact and save the content/body in the object (inline)
+    artifact = project.log_artifact(
+        "x", body="123", is_inline=True, artifact_path=str(assets_path())
+    )
```

Comment:
I guess this is also a way, although not the cleanest one, I would've probably create an artifact.yaml in the assets path and check that I am able to import it